app/javascript/components/public_health/PatientsTable.js

```diff
@@ -665,10 +667,12 @@ class PatientsTable extends React.Component {
                           <span className="ml-2">Close Records</span>
                         </Dropdown.Item>
                       )}
-                      <Dropdown.Item className="px-3" onClick={() => this.setState({ action: 'Update Case Status' })}>
-                        <i className="fas fa-clipboard-list text-center" style={{ width: '1em' }}></i>
-                        <span className="ml-2">Update Case Status</span>
-                      </Dropdown.Item>
+                      {this.props.workflow !== 'global' && (
```

Comment:
is the reasoning for not including this from the global view because we're assuming users would never update the case status of patients from different workflows to the same value? I'm just imagining what if there was a household where one individual started out in isolation while the others started out in exposure and they later on all tested positive or somehow all became confirmed cases.

Reply:
@hackrm and Kelly wanted this. Reasoning:
> the case status as you know is the data element that determines which workflow a monitroee will be in and that could get messy to deal on the global view if this type of update is done on multiple monitorees across multiple workflows.

Reply:
got it, thanks for the explanation